core/modules/replica/ConfigApp.cc

```diff
@@ -861,109 +1802,44 @@ int ConfigApp::_updateWorker() const {
 
     string const context = "ConfigApp::" + string(__func__) + "  ";
 
-    if (not _config->isKnownWorker(_workerInfo.name)) {
+    if (!_config->isKnownWorker(_workerInfo.name)) {
```

Comment:
Because it's not Python. At some, I thought it was a good idea to use `not` in place of `!`. Later, I realized it's very uncommon to use `not` and `and` in logical expression sin C/C++. Very few people do this.

Reply:
Did you mean to say `it's ! Python`? 😸 This is an odd argument. You have `not` in many other places, and this change looks like a random update at random location not even related to anything you do on this ticket.